app/models/sequenceCounterModel.js

```diff
@@ -1,7 +1,7 @@
 const mongoose = require('mongoose');
 
 const SequenceCounterSchema = new mongoose.Schema({
-  // _id: { type: String, required: true, unique: true }, // auto-increase Id
+  sequence_name: { type: String, required: true, unique: true }, // auto-increase Id
```

Comment:
Why using sequence_name instead of the default _id is a better idea?

Reply:
The new version of MongoDB does not allow changes to the _id.

Reply:
> The new version of MongoDB does not allow changes to the _id.

I've met the same problem too, was confused about not using '_id'. 
For my understanding, you don't have to define a 'sequence_name' if you just need a unique auto-increased id. You only need it when you want the auto-increased id to be interger or other types you want to define, cuz '_id' in Mongodb is an Object type and you need to access the actual id by '_id.ObjectId'.

Reply:
Plus: '_id' is auto generated by Mongodb

Reply:
> The new version of MongoDB does not allow changes to the _id.

Then in what condition that you need to change the default _id?